### PR TITLE
Update of EMBL hardware objects

### PR DESCRIPTION
**Pull request by IvarsKarpics**
* Updated EMBL hardware objects

### CODE REVIEW

**Comment by IvarsKarpics** [comment]
After some cleanup this PR is ready to be reviewed, merged. It also contains pylint check for EMBL hardware objects. At some point we should check also abstract, mockup and all other hardware objects with pylint.

File: HardwareObjects/EMBL/EMBLCRL.py [hunk 1]
```diff
@@ -81,16 +81,17 @@ def init(self):
...
-            for index in range(self.lens_count):
-                lens_combination = lens_combination + value[index] * pow(2, index)
-        else:
-            lens_combination = [0, 0, 0, 0, 0, 0]
-            for index in range(self.lens_count):
-                lens_combination[index] = (value & pow(2, index)) / pow(2, index)
-
-        return lens_combination
+        if value:
+            if type(value) in (list, tuple):
```
Comment (rhfogh) [inline HardwareObjects/EMBL/EMBLCRL.py]: Better:
if isinstance(value, (list, tuple)):

File: HardwareObjects/EMBL/EMBLPPUControl.py [hunk 2]
```diff
@@ -69,13 +69,14 @@ def all_restart_changed(self, status):
         self.restart_running = status
 
     def file_info_changed(self, value):
+        value = (value)
```
Comment (rhfogh) [inline HardwareObjects/EMBL/EMBLPPUControl.py]: Something rotten here:
1) 'value = (value)' is a no-op
2) if you meant 'value = (value,)' the length is always 1
3) if you are exrreacting a single value from a list or tuple, the list of tuple should be named 'values' (plural)

**Comment by rhfogh** [comment]
Has it been decided that all imports must be from the top level i.e. that HardwareRepository should be removed from the PythonPath)? I highly approve, I just want to know the exact state of play

**Comment by IvarsKarpics** [comment]
Hi @rhfogh , thank you for the useful suggestions! I will update my code.

**Comment by marcus-oscarsson** [comment]
@rhfogh Nothing have been decided yet but I think we basically all agree that its a good idea. It will most likely have some practical implications on how things are referenced from XML files and how that's going to be handled needs to be discussed.

I see no real reason for not using absolute imports, without appending additional paths to the python path, from python code we write.

**Comment by IvarsKarpics** [comment]
Pylint via travis indicated a typo on my code. Nice! we should definitely run pylint on all HardwareRepository.

**Comment by marcus-oscarsson** [comment]
Yes I agree, I was hoping that we eventually could run it without the -E option

**Pull request merged**